<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>741</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1010</width>
    <height>701</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1010</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cliptor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/icons/cliptor.png</normaloff>:/icons/cliptor.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frameA">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frameB">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCueA">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/icons/cue-set.png</normaloff>:/icons/cue-set.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPlayA">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="slideCrossfade">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>true</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSearch">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>380</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QSlider" name="slideVolumeA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>32</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listB">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>40</y>
      <width>161</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelDeckA">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deck A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDeckB">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deck B</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="slideVolumeB">
    <property name="geometry">
     <rect>
      <x>982</x>
      <y>290</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>32</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPlayB">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>290</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCueB">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>290</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/icons/cue-set.png</normaloff>:/icons/cue-set.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="slidePosA">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>321</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelPosA1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPosA2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="slidePosB">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>321</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelPosB2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>310</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPosB1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableResults">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>431</y>
      <width>991</width>
      <height>261</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>vid</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>author</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>published</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>desc</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>length</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>rating</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>views</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>thumbs</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="buttonRecommend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Match</string>
    </property>
   </widget>
   <widget class="QSlider" name="slideBPMA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelBPMA">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>000 BPM</string>
    </property>
   </widget>
   <widget class="QSlider" name="slideCrossfadeBPMB">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>340</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelBPMB">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>340</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>000 BPM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="editSearch">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listSearch">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>421</y>
      <width>531</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/icons/cliptor16.png</normaloff>:/icons/cliptor16.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>editSearch</tabstop>
  <tabstop>listSearch</tabstop>
  <tabstop>buttonSearch</tabstop>
  <tabstop>buttonRecommend</tabstop>
  <tabstop>tableResults</tabstop>
  <tabstop>listA</tabstop>
  <tabstop>listB</tabstop>
  <tabstop>slideVolumeA</tabstop>
  <tabstop>buttonCueA</tabstop>
  <tabstop>buttonPlayA</tabstop>
  <tabstop>slidePosA</tabstop>
  <tabstop>slideBPMA</tabstop>
  <tabstop>slidePosB</tabstop>
  <tabstop>buttonPlayB</tabstop>
  <tabstop>buttonCueB</tabstop>
  <tabstop>slideVolumeB</tabstop>
  <tabstop>slideCrossfadeBPMB</tabstop>
  <tabstop>slideCrossfade</tabstop>
 </tabstops>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
